style.css

```diff
@@ -0,0 +1,166 @@
+body {
+  margin: 20px;
+  margin-left: 35px;
+  text-align: center;
+  color: black;
+  scroll-behavior: smooth;
+}
+a:link, a:visited {
+  text-decoration: none;
+  color: black;
+  font-size: 25px;
+  font-family: permanent marker;
+}
+
+.alphabet{
+  font-size: 10px;
+  line-height: 120%;
+}
+p{
+  font-family: peddana;
+  width: 250px;
+  margin: 10px auto 10px auto;
+  line-height: 80%;
+  font-size: 25px;
+}
+h1 { 
+    margin-top: 15px;
+    margin-left: 15px;
+    margin-right: 15px;
+    margin-bottom: 15px;
+    padding-top: 20px;
+    font-size: 35px;
+    line-height: 80%;
+    height: auto;
+    text-align: center;
+    font-family: permanent marker;
+}
+h2{
+  font-size: 20px;
+  font-family: permanent marker;
+}
+h3{
+  font-size: 60px;
+  margin: 0px;
+  text-align: center;
+  vertical-align: middle;
+  color: rgb(252, 77, 15);
+}
+.flex-parent{
+  justify-content: center;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  font-family: peddana;
+  align-items: flex-end;
+  border-left-style: solid;
+  border-left-width: 6px;
+  border-left-color: black;
+}
+.flex-child{
+  margin: 10px;
+  width: 250px;
+  height: auto;
+  padding-left: 15px;
+  padding: 10px;
+  padding-bottom: 10px;
+  text-align: center;
+  font-size: 15px;
+  font-family: permanent marker;
+}
+.a,.b,.c,.d,.e,.f,.g,.h,.i,.J,.k,.l,.m,.n,.o,.p,.q,.r,.s,.t,.u,.v,.w,.x,.y,.z{
```

Comment:
I can't see any elements with these class names. You could apply styling to the flex-children by targeting `.flex-child`